clientlibrary/worker/worker.go

```diff
@@ -194,6 +203,15 @@ func (w *Worker) initialize() error {
 		log.Infof("Use custom checkpointer implementation.")
 	}
 
+	if w.kclConfig.EnableLeaseStealing {
```

Comment:
It should be just part of the checkpointer and no need to inject the lease stealer.

Reply:
Fixed